docs/source/beersheba.rst

```diff
@@ -251,6 +254,60 @@ Deconvolution process finishes when any of the stopping criteria occurs:
 RL deconvolution is implemented in Beersheba using the Richardson-Lucy function from Python’s scikit-image library. Additional
 details about how the RL deconvolution works and the way the PSFs are implemented can be found in the RL publication [#]_.
 
+.. _Satellite-Killer:
+
+Satellite Killer
+:::::::::::::::::
+
+A common issue within the deconvolution process is the creation of 'satellite tracks'.
+These are small artificial [#]_ energy deposits that exceed the later applied cleaning cut, generated
+due to energy that is unable to be deconvolved back to it's initial point and instead coalescing
+around the main track (hence satellites). This can be seen in the plot below on the left.
+
+**Satellite killer** is an algorithm introduced to remove these satellites *during* the deconvolution
+process, allowing for a resulting track that contains no artificial hits as seen on plot below on the right. 
+It is fine-tunable, allowing the end user to change the strength of the implementation easily.
+
+.. image:: images/beersheba/satellites.png
+  :width: 100%
+
+These parameters are stored within the dictionary ``satellite_params``, and described below:
+
+.. |sk| raw:: html
+
+     <sk>
```

Comment:
what's this?

Reply:
Similar to previously, I just copied the format for the other parameter table in the beersheba doc.
I don't know Sphinx all too well, but based on what I've seen I assumed it was some manner of allowing for references to the table for ease of access. I'll remove it now, as it isn't referenced anywhere.